pkg/aws/aws.go

```diff
@@ -80,6 +85,38 @@ func NewProvider(ctx context.Context, logs log.Logger) (*AwsProvider, error) {
 	return provider, nil
 }
 
+func NewAWSConfig(ctx context.Context, logs log.Logger, options *options.Options) (aws.Config, error) {
+	var opts []func(*awsConfig.LoadOptions) error
+	if options.CustomCredentialCommand != "" {
+		var output bytes.Buffer
+		cmd := exec.Command("sh", "-c", options.CustomCredentialCommand)
+		cmd.Stdout = &output
+		cmd.Stderr = logs.Writer(logrus.ErrorLevel, true)
+		if err := cmd.Run(); err != nil {
+			return aws.Config{}, fmt.Errorf("run command %q: %w", options.CustomCredentialCommand, err)
+		}
+
+		// parse the JSON output to an aws.Credentials object
+		var creds aws.Credentials
+		if err := json.Unmarshal(output.Bytes(), &creds); err != nil {
+			return aws.Config{}, fmt.Errorf("parse JSON output %q: %w", output.Bytes(), err)
```

Comment:
could you update the log to say `parse aws credential JSON output`

Reply:
sure, done